docs/management/advanced-options.asciidoc

```diff
@@ -286,20 +286,23 @@ in the current data view is used.
 The columns that appear by default on the *Discover* page. The default is
 `_source`.
 
-[[discover-samplesize]]`discover:sampleSize`::
-The number of rows to show in the *Discover* table.
+[[discover:enableSql]]`discover:enableSql`::
+experimental[] Allows SQL queries for search.
 
 [[discover-max-doc-fields-displayed]]`discover:maxDocFieldsDisplayed`::
 Specifies the maximum number of fields to show in the document column of the *Discover* table.
 
 [[discover-modify-columns-on-switch]]`discover:modifyColumnsOnSwitch`::
 When enabled, removes the columns that are not in the new data view.
 
-[[discover-sample-size]]`discover:sampleSize`::
-Specifies the number of rows to display in the *Discover* table.
+[[discover-row-height-option]]`discover:rowHeightOption`::
+The number of lines to allow in a row. A value of -1 automatically adjusts the row height to fit the contents. A value of 0 displays the content in a single line.
 
 [[discover-sampleRowsPerPage]]`discover:sampleRowsPerPage`::
-Specifies the number of rows to display per page in the *Discover* table.
+Limits the number of rows per page in the document table.
+
+[[discover-sample-size]]`discover:sampleSize`::
+Sets the maximum number of rows for the entire document table.
```

Comment:
It's a technical detail, but I think it's worth to mention, that this is the maximum number of documents fetched from Elasticsearch